### PR TITLE
Fix some broken links

### PR DESCRIPTION
**Pull request by eloycoto**
Signed-off-by: Eloy Coto <eloy.coto@acalustra.com>

### CODE REVIEW

File: assets/nav.scss [hunk 1]
```diff
@@ -77,7 +77,7 @@
   position: absolute;
   padding-top: 10px;
   left: 109px;
-  width: 60px;
+  width: 160px;
```
Comment (gciavarrini) [inline assets/nav.scss:80]: IMO it can be even bigger

**Review by gciavarrini — APPROVED** [review]

File: assets/nav.scss (hunk 1 above)
Comment (eloycoto) [reply assets/nav.scss:80]: make the font bigger :joy:

**Review by jakub-dzon — APPROVED** [review]
Nice. Thank you!

**Comment by eloycoto** [comment]
> By deleting this, the navigation bar does not work anymore as the path are concatenated (i.e: http://localhost:3000/documentation/blog if you select Documentation then blog

Umm, but works from index that was not working at all :\ checking

**Pull request merged**